src/main/java/com/google/sps/data/DatastorePersonDao.java

```diff
@@ -44,7 +44,15 @@ public DatastorePersonDao() {
    * We make an entity in Datastore with person's fields as properties and their email as the key.
    */
   @Override
-  public void put(Person person) {
+  public void create(Person person) {
+    datastore.put(personToEntity(person));
+  }
+
+  /**
+   * We update an entity in Datastore with person's fields as properties and their email as the key.
+   */
+  @Override
+  public void update(Person person) {
```

Comment:
How is this updating? Does it replace the old one because they both have the same keys (email) ?

Reply:
Yes, exactly.